src/seedu/addressbook/AddressBook.java

```diff
@@ -526,6 +537,62 @@ private static boolean isDeletePersonArgsValid(String rawArgs) {
             return false;
         }
     }
+    
+    /**
+     * Edits the phone number of a person identified using last displayed index.
+     *
+     * @param commandArgs full command args string from the user
+     * @return feedback display message for the operation result
+     */
+    private static String executeEditPhone(String commandArgs) {
+        String[] splitUserArgs = commandArgs.split("\\s+");
+
+        // check if there are the correct number of arguments in the entered command
+        if (splitUserArgs.length != 2) {
+            return getMessageForInvalidCommandInput(COMMAND_EDIT_PHONE_WORD, getUsageInfoForEditPhoneCommand());
+        }
+        
+        // checks if the index entered is a number
+        if (!isEditPhoneArgsValid(splitUserArgs[0])) {
+            return getMessageForInvalidCommandInput(COMMAND_EDIT_PHONE_WORD, getUsageInfoForEditPhoneCommand());
+        }
+
+        // checks if the phone number entered is valid
+        final String phoneNumberToChange = splitUserArgs[1].trim();
+        if (!phoneNumberToChange.matches("[0-9]+")) {
+            return getMessageForInvalidCommandInput(COMMAND_EDIT_PHONE_WORD, getUsageInfoForEditPhoneCommand());
+        }
```

Comment:
A lot of checks above here which could have been pulled out into a function that returns boolean, e.g. do `if(!isEditPhoneArgsValid(commandArgs) return getMessage...`, refer to the other commands. This has the advantage that you can change the behaviour when receiving invalid input to something else more easily.